modules/upgrade/pages/client-intro.adoc

```diff
@@ -0,0 +1,10 @@
+[[client-migration]]
+= Upgrade the Clients
+
+
+Clients use the versioning system of their underlying operating system.
+For most clients using {suse} operating systems, you can perform minor upgrade within the {productname} {webui}.
```

Comment:
I think we can safely say:
```suggestion
For clients using {suse} operating systems, you can perform upgrades within the {productname} {webui}.
```
This would cover package and SP upgrades.